app/src/main/java/com/washingtondcsquad/tudee/presentation/components/SnackBarCard.kt

```diff
@@ -0,0 +1,77 @@
+package com.washingtondcsquad.tudee.presentation.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import com.washingtondcsquad.tudee.R
+import com.washingtondcsquad.tudee.presentation.design.AppTheme
+import com.washingtondcsquad.tudee.presentation.utils.modifierExensions.dropShadow
+
+
+@Composable
+fun SnackBarCard(
+    modifier: Modifier = Modifier,
+    isError: Boolean,
+    message: String
```

Comment:
we can make it more reusable by making it take icon and message ,
think of it if we have a **warring snackbar**